testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/two-kerberos-hives/presto-krb5.conf

```diff
@@ -8,7 +8,8 @@
  dns_lookup_realm = false
  dns_lookup_kdc = false
  ticket_lifetime = 24h
- renew_lifetime = 7d
+ # this setting is causing a Message stream modified (41) error when talking to KDC running on CentOS 7: https://stackoverflow.com/a/60978520
```

Comment:
that is the symptom, not the cause - see https://github.com/trinodb/docker-images/pull/162

Reply:
Allowing principals to renew tickets doesn't seem to solve it. I'll keep these changes, and we can try fixing this in a separate PR.

Reply:
Thanks for following up.

Can you file an issue with the logs since I think we'll lose the logs eventually and then no-one will be motivated to look into this.